modules/auxiliary/scanner/mysql/mysql_hashdump.rb

```diff
@@ -17,16 +17,16 @@ def initialize
         hashes from a MySQL server and stores them for later cracking.
       ),
       'Author'         => ['theLightCosine'],
-      'License'        => MSF_LICENSE
+      'License'        => MSF_LICENSE,
+      'SessionTypes'  => %w[MySQL]
     )
   end
 
   def run_host(ip)
-
     return unless mysql_login_datastore
 
     service_data = {
-      address: ip,
+      address: rhost,
```

Comment:
Why is this change needed?

Reply:
It wasn't needed, it was leftover from when I was testing something and forgot to revert it. Good catch 👍